pkg/lib/cockpit-components-terminal.jsx

```diff
@@ -191,6 +198,18 @@ export class Terminal extends React.Component {
     render() {
         return (
             <>
+                <Modal title={_("Pasting in cockpit terminal")}
```

Comment:
Please no "cockpit". Maybe "Paste error"?

Reply:
Simple and good. Yes, I always forget that cockpit is not a term not RHEL.